src/components/Image.js

```diff
@@ -5,8 +5,29 @@ import { css } from '@emotion/core';
 import { getNestedValue } from '../utils/get-nested-value';
 import { size } from './dev-hub/theme';
 
+const getAlignment = align => {
```

Comment:
Can we also add this to the `Chart` component? There is similar logic there and they likely will want the same behavior between charts and images.